src/main/java/io/codearte/jfairy/Bootstrap.java

```diff
@@ -147,6 +143,8 @@ private static FairyModule getFairyModuleForLocale(DataMaster dataMaster, Locale
 				return new ZhFairyModule(dataMaster, randomGenerator);
 			case DE:
 				return new DeFairyModule(dataMaster, randomGenerator);
+			case KA:
```

Comment:
Is the language code GE or KA?

Reply:
As mentioned in the PR description, this a rare case where a name of the country and language differs. 'GE' is the country (ISO 3166) , 'KA' is the language (https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=ka).

Reply:
Ok. Looking back at it, the way it's done right now is inconsistent. We use language codes, while the fairy modules really represent specific locale, not only languages. I think you can keep KA here, but soon we should  modify these enums to represent both language and country, cause there can be different document formats in same language speaking countries.